.github/cilium-actions-master.yml

```diff
@@ -0,0 +1,37 @@
+project: "https://github.com/cilium/cilium/projects/80"
+column: "In progress"
+move-to-projects-for-labels-xored:
+  v1.6:
+    needs-backport/1.6:
+      project: "https://github.com/cilium/cilium/projects/91"
```

Comment:
Is the plan to bump these project links every single time we do a release?

Reply:
Hrm, given that you've marked this PR for backport, I'm getting the impression that GH reads these from each branch, so I'm a bit confused which of these projects links we will need to update each time we make a new release. Would be good to see that process written down somewhere.

Reply:
`.github/cilium-actions-master.yml` should be in master, i.e., this PR, all the other files should be in their respecting branch. It is correct, once we release a new version of 1.6, we need to bump update these projects.